modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -40,22 +40,27 @@ A cluster might function with fewer than recommended resources, but its performa
 If OpenStack Object Storage (Swift) is available and operated by a user account with the `swiftoperator` role, it is used as the default backend for the {product-title} image registry. In this case, the volume storage requirement is 175 GB. Swift space requirements vary depending on the size of the image registry.
 ====
 
-A cluster might function with fewer than recommended resources, but its
-performance is not guaranteed.
+[IMPORTANT]
+====
+If you are using {rh-openstack-first} version 16 with the Amphora driver rather than the OVN Octavia driver, security groups are associated with Service accounts instead of user projects.
+====
 
 Take the following notes into consideration when setting resources:
 
-* The number of ports required is actually larger than the number of Pods. Kuryr
+* The number of ports that are required is larger than the number of Pods. Kuryr
 uses ports pools to have pre-created ports ready to be used by Pods and speed up
-the Pods booting time.
+the Pods' booting time.
 
 * Each NetworkPolicy is mapped into an {rh-openstack} security group, and
 depending on the NetworkPolicy spec, one or more rules are added to the
 security group.
 
-* Each Service is mapped into an {rh-openstack} load balancer. Each load balancer
-has a security group with the user project; therefore, it must be taken into
-account when estimating the number of security groups required for the quota.
+* Each Service is mapped to an {rh-openstack} load balancer. Consider this requirement
+ when estimating the number of security groups required for the quota.
++
+If you are using
+{rh-openstack} versions 15 and earlier, or the `ovn-octavia driver`, each load balancer
```

Comment:
s/versions 15 and earlier/version 15 or earlier/
Unless they could be running both 15 and earlier?